public/assets/scss/header/_header.scss

```diff
@@ -269,18 +269,19 @@
                     }
                 }
             }
-            .humberger-menu {
+            .hamburger-menu {
                 span {
                     &.text-white {
                         color: $heading-color !important;
+                        color: red !important;
```

Comment:
What is this line for? (ln:276)

Reply:
BTW, it is fine to leave off the `Advice for Reviewers & Testing Notes` and `Screenshots` sections for simple stories, but if doing so please delete those sections before submitting